torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3093,32 +3093,49 @@ def error_inputs_max_pool1d(op_info, device, **kwargs):
                      error_regex='pad should be at most half of kernel size')
 
     # error inputs for input tensor
-    yield ErrorInput(SampleInput(torch.tensor(0), kwargs={'kernel_size': 1}),
-                     error_regex='Expected 2D or 3D input tensor, but got')
+    yield ErrorInput(SampleInput(torch.tensor(0, device=device), kwargs={'kernel_size': 1}),
+                     error_regex=r'Expected 2D or 3D \(batch mode\) tensor with optional')
 
     # error inputs for empty input
-    yield ErrorInput(SampleInput(torch.tensor([]), kwargs={'kernel_size': 1}),
-                     error_regex='Expected 2D or 3D input tensor, but got')
+    yield ErrorInput(SampleInput(torch.tensor([], device=device), kwargs={'kernel_size': 1}),
+                     error_regex=r'Expected 2D or 3D \(batch mode\) tensor with optional')
 
     # error inputs for empty input with stride=0
-    yield ErrorInput(SampleInput(torch.tensor([[]]), kwargs={'kernel_size': 1, 'stride': 0}),
-                     error_regex='stride must be greater than zero, but got 0')
+    # NOTE: CPU and CUDA error messages are different
```

Comment:
Should this PR make them consistent?

Reply:
I think it should be done in another PR. Will open an issue for the same.

Reply:
Issue: https://github.com/pytorch/pytorch/issues/85712